src/utils/common.js

```diff
@@ -39,6 +42,8 @@ export const hasTruthyValue = (value) => {
   return values.every(item => !!item);
 };
 
+export const sumOfArray = (values) => values.reduce((prev, next) => prev + next, 0);
```

Comment:
[sanity check] Verify `sumOfArray` is used with (coerced?) numbers before adding.